run_aprime.bash

```diff
@@ -395,6 +415,17 @@ else
   export mpasAutocloseFileLimitFraction=0.2 # default value
 fi
 
+if [ -f $test_archive_dir/$test_casename/run/mpas-cice_in ]; then
+  export seaIce_namelist_file=mpas-cice_in
+else
+  export seaIce_namelist_file=mpas-seaice_in
+fi
```

Comment:
I would switch these.  If both exist (for some reason) we would want to go with the new name over the old (e.g. for testing).  The same with the streams files.